code_samples/paging_predicate_sample/README.md

```diff
@@ -0,0 +1,23 @@
+# Paging Predicate Example with Sorting
+
+In this example you will learn how you can use a paging predicate with sorting. In order to define how sorting will work,
+you need a comparator. In order a member to know this comparator, you need to register the comparator in the member config.
+The following describes how to do it:
+
+You can use the sample project in [main](./main) folder:
+
+1. Run the [Main](./main/src/main/java/Main.java) class. The started member will already have the `Comparator` class registered.
+2. Run the sample code in main.js via `node main.js`.
+
+Alternatively, you can start from scratch:
+
+1. Download hazelcast. Extract it somewhere. https://hazelcast.org/imdg/download/
+2. To add a custom comparator you need to compile and add it to hazelcast classpath.
+3. In order to compile use a java project that includes hazelcast as a dependency, you can use an IDE like Intellij for simplicity. Just open the IDE and press build button, then the classes will be in `target` folder. The java project used is in the `main` folder.
```

Comment:
```suggestion
3. In order to compile use a Java project that includes hazelcast as a dependency, you can use an IDE like Intellij for simplicity. Just open the IDE and press build button, then the classes will be in `target` folder. The java project used is in the `main` folder.
```
Can you do the same for other instances?

Also, `Intellij` is not the name of the IDE, it is `IDEA` or `Intellij IDEA`